dist/client.js

```diff
@@ -459,7 +459,6 @@
     define([], function() {
       return CrossStorageClient;
     });
-  } else {
-    root.CrossStorageClient = CrossStorageClient;
   }
+  root.CrossStorageClient = CrossStorageClient;
```

Comment:
And sidenote, if you don't want to maintain a fork, this solution would force the client to populate the global object: https://github.com/electron/electron/issues/254#issuecomment-208044147